yarn.lock

```diff
@@ -1,7 +1,5 @@
 # THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
@armstrongb I dont think we are supposed to commit the `yarn.lock` are we?

Reply:
https://yarnpkg.com/lang/en/docs/yarn-lock/#toc-check-into-source-control

Reply:
I stand corrected. Thanks @benjefferies